templates/todoList.html

```diff
@@ -4,10 +4,10 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>To-Do List Component</title>
-  <link rel="stylesheet" href="{{ url_for('static', filename='css/todo.css') }}">
+  <link rel="stylesheet" href="{{ url_for('static', filename='css/todo.css') }}?v={{ time.time() }}">
```

Comment:
what's the reason for the time code here?

Reply:
oops left that in on accident. That was just so that I could view changes in real time (added that at the very beginning but I don't need it anymore so I'll delete it) -- good catch!